Framework/Core/src/CommonDataProcessors.cxx

```diff
@@ -354,7 +354,7 @@ DataProcessorSpec
         auto s = pc.inputs().get<TableConsumer>(ref.spec->binding);
         auto table = s->asArrowTable();
         if (!table->Validate().ok()) {
-          LOGP(WARNING, "The table \"{}\" is not valid and will not be saved!", tableName);
+          LOGP(warnING, "The table \"{}\" is not valid and will not be saved!", tableName);
```

Comment:
is that a typo? `LOGP(warn)` ?

Reply:
It is actually a feature of the upper to lower case converter: it checks only for `|LOGP[(]WARN|LOGP(warn|g;`
@davidrohr can this be fixed?

Reply:
@ktf I've learned that this case converter is yours, can you add `WARNING`?